shared/src/sql/index_setup.sql

```diff
@@ -0,0 +1,40 @@
+CREATE TABLE one_republic_songs (
```

Comment:
if you're using this exclusively in the tests, it could go in the `pg_bm25/test/fixtures.sql` file instead

Reply:
But this is just for the unit tests, not the regression tests.

Reply:
Good point. Will defer to @neilyio or @rebasedming for their thoughts

Reply:
Yep this is okay, I've done the same thing in the `shared` crate. The relative import is much simpler for unit tests.